src/test/java/seedu/plannermd/model/patient/RiskTest.java

```diff
@@ -22,14 +22,14 @@ public void constructor_invalidRisk_throwsIllegalArgumentException() {
 
     @Test
     public void isValidRisk() {
-        // null address
+        // null risk
         assertThrows(NullPointerException.class, () -> Risk.isValidRisk(null));
 
-        // invalid addresses
+        // invalid risks
         assertFalse(Risk.isValidRisk("")); // empty string
         assertFalse(Risk.isValidRisk(" ")); // spaces only
 
-        // valid addresses
+        // valid risks
         assertTrue(Risk.isValidRisk("HIGH"));
         assertTrue(Risk.isValidRisk("MEDIUM")); // one character
         assertTrue(Risk.isValidRisk("LOW")); // long address
```

Comment:
Perhaps can remove the comments too as they are not related to risk.
```suggestion
        assertTrue(Risk.isValidRisk("MEDIUM"));
        assertTrue(Risk.isValidRisk("LOW"));
```